test/components/live_redirect_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule Surface.Components.LiveRedirectTest do
+  use ExUnit.Case, async: true
+
+  alias Surface.Components.LiveRedirect, warn: false
+
+  import ComponentTestHelper
+
+  defmodule ComponentWithLink do
+    use Surface.LiveComponent
+
+    def render(assigns) do
+      ~H"""
+      <div>
+        <LiveRedirect to="/users/1"/>
+      </div>
+      """
+    end
+
+    def handle_event(_, _, socket) do
+      {:noreply, socket}
+    end
+  end
+
+  describe "Without LiveView" do
```

Comment:
I think those `describe "Without LiveView"` that we have in many test cases don't make sense anymore. We should stop adding them to new tests and later remove them from the other tests.